test/test_pmu.py

```diff
@@ -44,3 +44,131 @@ def test_interp_resp_trace():
 
     assert(np.all(np.isclose(acq_pressure[index_pmu_interp], pmu.data[index_pmu_data], atol=1, rtol=0.08)))
 
+
+from matplotlib.figure import Figure
+from shimmingtoolbox import __dir_shimmingtoolbox__
+import nibabel as nib
+import json
+from shimmingtoolbox.masking.shapes import shapes
+
+
+def test_timing_images():
+    """Check the matching of timing between MR images and PMU timestamps"""
+
+    # Convert to ms
+    def dicom_times_to_ms(dicom_times):
```

Comment:
that looks like something that could go in an API. Not sure where would make more sense.

Reply:
That's why it is still in the test, my first thought was in /shimmingtoolbol/utils.py but it does not really fit with what is currently in there. Another option is in pmu.py but the file is already a class so I would prefer not mix a class and a standalone function.

Reply:
Related to #151

Reply:
Function moved to /utils